smokes/master.cfg

```diff
@@ -129,4 +129,3 @@ c['www']['auth'] = auth
 c['www']['authz'] = authz
 
 # in order to share this snippet in the doc, we load mydashboard.py using exec
-exec(open("mydashboard.py").read())
```

Comment:
I think we have one test that checks whether the dashboard example works.

Reply:
arg, I did a commit -a  by mistake!